Formula/glib.rb

```diff
@@ -31,6 +32,10 @@ def install
     inreplace %w[gio/gdbusprivate.c gio/xdgmime/xdgmime.c glib/gutils.c],
       "@@HOMEBREW_PREFIX@@", HOMEBREW_PREFIX
 
+    inreplace "meson.build",
+      "python_name = 'python3'",
+      "python_name = '#{Formula["python@3.8"].opt_bin}/python3'"
```

Comment:
I am seeing, in the first few PRs to migrate things over, there seems to be a need in several places to hack the python3.8 path. And in other cases, the python path / executable name has to be changed by inreplace in all generated scripts/binaries. Which I think would be considered Really Bad by our usual rules 😞 

Can we find a generic solution to this issue that does not involve hacking scripts/build systems?

Reply:
Ok, here is my next attempt with `write_env_script` 👀